File: Solution/CSharp/217.ContainsDuplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        int length = nums.Length;
        int num = 0;
        Dictionary<int, int> d = new Dictionary<int, int>();
        for (int i = 0; i < length; i++) {
            num = nums[i];
            if (d.TryGetValue(num, out value)) {
                return true;
            } else {
                d[num] = 1;
            }
            return false;
        }
    }

File: Solution/Javascript/217. Contains Duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let m = {};
    for(let i = 0; i < nums.length; i++){
        let currentNum = nums[i];
        if(!m[currentNum]){
            m[currentNum] = true
        }else{
            return true;
        }
    }
    return false;
};